src/redux/reducers/videosReducer.js

```diff
@@ -0,0 +1,49 @@
+import { storage } from '../../utils/storage';
```

Comment:
What does storages does? I don't understand why you call storage.set('videos', JSON.stringify(something)); on every case for the reducer